.travis.yml

```diff
@@ -24,6 +24,7 @@ cache:
 env:
   - ROS_RELEASE=kinetic DOCKER_CACHE_FILE=/home/travis/docker/kinetic-cache.tar.gz
   - ROS_RELEASE=melodic DOCKER_CACHE_FILE=/home/travis/docker/melodic-cache.tar.gz
+  - ROS_RELEASE=noetic DOCKER_CACHE_FILE=/home/travis/docker/noetic-cache.tar.gz
```

Comment:
Noetic CI requires changes in cartographer (python3-sphinx etc), this should fix it: https://github.com/cartographer-project/cartographer/pull/1705